src/hooks/use-focus.ts

```diff
@@ -12,13 +12,23 @@ interface Input {
 	 * Auto focus this component, if there's no active (focused) component right now.
 	 */
 	autoFocus?: boolean;
+
+	/**
+	 * Assign an ID to this component, so it can be programmatically focused with `focus(id)`.
```

Comment:
```suggestion
	 * Assign an ID to this component, so it can be programmatically focused with `.focus(id)`.
```

Reply:
It's not an instance method. This is the callback returned from `useFocus` and `useFocusManager`.